internal/db/storagetype_string.go

```diff
@@ -0,0 +1,25 @@
+// Code generated by "stringer -type=StorageType -trimprefix=StorageType"; DO NOT EDIT.
+
+package db
+
+import "strconv"
+
+func _() {
+	// An "invalid array index" compiler error signifies that the constant values have changed.
+	// Re-run the stringer command to generate them again.
+	var x [1]struct{}
+	_ = x[storageTypeUnknown-0]
+	_ = x[StorageTypeMongo-1]
+	_ = x[storageTypeSentinel-2]
+}
+
+const _StorageType_name = "storageTypeUnknownMongostorageTypeSentinel"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [don't use underscores in Go names; const _StorageType_name should be _StorageTypeName](https://app.codacy.com/app/oleg.balunenko/logs-converter/pullRequest?prid=4555937)